Gopkg.lock

```diff
@@ -4,20 +4,28 @@
 [[projects]]
   name = "cloud.google.com/go"
   packages = ["compute/metadata"]
-  revision = "767c40d6a2e058483c25fa193e963a22da17236d"
-  version = "v0.18.0"
+  revision = "050b16d2314d5fc3d4c9a51e4cd5c7468e77f162"
+  version = "v0.17.0"
```

Comment:
how come we're rolling back off these pkg versions? Would you mind running a `dep ensure` if you haven't already?

Reply:
I think this was related to reverting the helm bump commit, not entirely sure TBH :) Once https://github.com/Azure/draft/pull/453 is merged however we should be able to rebase this PR and avoid any changes to Gopkg files.